MapboxNavigation/TopBannerViewController.swift

```diff
@@ -97,13 +97,19 @@ open class TopBannerViewController: UIViewController {
         return view
     }()
     
+    lazy var junctionView: JunctionView = {
+        let view: JunctionView = .forAutoLayout()
+        view.isHidden = true
+        return view
+    }()
+    
     private let instructionsBannerHeight: CGFloat = 100.0
     
     private var informationChildren: [UIView] {
         return [instructionsBannerView] + secondaryChildren
     }
     private var secondaryChildren: [UIView] {
-        return [lanesView, nextBannerView, statusView]
+        return [lanesView, nextBannerView, statusView, junctionView]
```

Comment:
#2434 tracks hooking up a JunctionView when guidance cards are enabled as an alternative to TopBannerViewController.